test/parallel/test-https-host-headers.js

```diff
@@ -8,8 +10,8 @@ const https = require('https');
 const fs = require('fs');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readSync('keys/agent1-key.pem'),
+  cert: fixtures.readSync('keys/agent1-cert.pem')
```

Comment:
Suggest using `fixtures.readKey` instead of `fixtures.readSync` for fixtures in `keys`.